codegen/src/main/scala/akka/grpc/gen/scaladsl/ScalaCodeGenerator.scala

```diff
@@ -69,3 +62,11 @@ abstract class ScalaCodeGenerator extends CodeGenerator {
     }
   }
 }
+object ScalaCodeGenerator {
+  val generateServiceFile: Service => CodeGeneratorResponse.File = service => {
+    val b = CodeGeneratorResponse.File.newBuilder()
+    b.setContent(ApiTrait(service).body)
+    b.setName(s"${service.packageName.replace('.', '/')}/${service.name}.scala")
```

Comment:
Time for some util for `packageName.replace('.', '/')`?

Reply:
It has that already actually `def packageDir = packageName.replace('.', '/')`

Reply:
Just added it in response to your comment ;)